Source/CombatExtended/CombatExtended/StatWorkers/StatWorker_ArmorPartial.cs

```diff
@@ -146,6 +146,73 @@ public override string GetExplanationFinalizePart(StatRequest req, ToStringNumbe
             return base.GetExplanationFinalizePart(req, numberSense, finalVal);
         }
 
+	public override string GetStatDrawEntryLabel(StatDef stat, float value, ToStringNumberSense numberSense, StatRequest optionalReq, bool finalized = true)
+        {
+	    if (this.stat == global::RimWorld.StatDefOf.ArmorRating_Blunt ||
+		    this.stat == global::RimWorld.StatDefOf.ArmorRating_Sharp) {
+
+		if(optionalReq != null) {
+		    if (optionalReq.Thing is Apparel apparel) {

```

Comment:
```suggestion
	    if (optionalReq?.Thing is Apparel apparel) {

```

Reply:
This null check protects both the contained `if`s, so you'd have to repeat it on line 180.